src/collector/poller.rs

```diff
@@ -109,6 +149,28 @@ impl<T: 'static + Middleware> Poller<T> {
             Ok(())
         } else {
             tick_info.reverse();
+            // TODO(Levi) needs to be initialized (should it be derived from the cosmos_key??)
+            let delegate_cosmos_address =
```

Comment:
Should be derived from the cosmos key that gets passed to the poller constructor

Reply:
Done https://github.com/PeggyJV/cellar_rebalancer_rs/blob/c3f93f33563adcd63ff450bf84384f464b67a8ec/src/collector/poller.rs#L155